PokemonGo/RocketAPI/Window/PokeUi.cs

```diff
@@ -96,8 +97,8 @@ private void PokeUi_Load(object sender, EventArgs e)
 
 
                     this.listView1.Items.Add(listViewItem);
-
-
+		    this.Text = "PokeUi " + countpkmn + "/" + profile.Profile.PokeStorage;
+                    countpkmn++;
```

Comment:
yeah actually without show the current increment on the title it's usless this variable changed with pokemons.Count<PokemonData>()